src/mix.ts

```diff
@@ -99,6 +176,16 @@ const completionSpec: Fig.Spec = {
   ],
 };
 
+function generatePreferredFilepaths(): Fig.Generator {
+  return {
+    template: "filepaths",
+    filterTemplateSuggestions: (paths) =>
+      paths.map((path) =>
+        path.name.endsWith(".exs") ? { ...path, priority: 75 } : path
+      ),
+  };
+}
+
```

Comment:
This doesn't need to be a function - it can just be a constant.
```suggestion
const preferredFilepaths: Fig.Generator = {
    template: "filepaths",
    filterTemplateSuggestions: (paths) =>
      paths.map((path) =>
        path.name.endsWith(".exs") ? { ...path, priority: 75 } : path
      ),
}

```

Reply:
It can be also simplified with the `@fig/autocomplete-generators` package!

Reply:
Thanks, I didn't see this either 🤦. I've simplified it based on what's in the elixir autocomplete using `@fig/autocomplete-generators` as suggested 🙏